### DISCUSSION TITLE
Speed

### DISCUSSION THREAD

**Discussion by veasel123** [12joan/twitter-client#41] [Q&A]
How fast can this fetch new tweets through RSS?

**Comment by 12joan** [comment]
I haven't tested it, but I would assume the API has access to Tweets as soon as they're posted.

As for how soon your RSS client will display them, that depends on the client. RSS is a pull-based protocol, so it fetches new content when the client requests it, not when the server pushes it.

**Reply by 12joan** [reply]
If you're asking about performance, it should be extremely fast. If everything is cached, it's only one API request from 12joan/twitter-client to Twitter to fetch a new batch of Tweets.

**Comment by veasel123** [comment]
What about rate limitations? Lets say you want to make sure you receive new tweets sub 200ms

**Reply by 12joan** [reply]
I haven't checked what the rate limits are, but it's possible that Twitter would denylist your IP address if you were making five requests per second for a prolonged length of time. But perhaps not; I expect public Nitter instances experience far greater traffic volumes without incident.

If you decide to try it out, let us know how it goes.

**Reply by 12joan** [reply]
My instance has been sending roughly one request every 8 seconds for the past few weeks without incident.
